core/src/executor/update.rs

```diff
@@ -4,8 +4,9 @@ use {
         evaluate::{evaluate, Evaluated},
     },
     crate::{
-        ast::{Assignment, ColumnDef, ColumnUniqueOption},
+        ast::{Assignment, ColumnDef, ColumnUniqueOption, ForeignKey},
         data::{Row, Value},
+        prelude::Key,
```

Comment:
we may not need to use `prelude` in internal codes. we can use `data::Key` instead